dupedetection/service.go

```diff
@@ -0,0 +1,279 @@
+package dupedetection
+
+import (
+	"bytes"
+	"context"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/btcsuite/btcutil/base58"
+	"github.com/pastelnetwork/gonode/common/errors"
+	"github.com/pastelnetwork/gonode/common/log"
+	"github.com/pastelnetwork/gonode/metadb/rqlite/db"
+	"github.com/pastelnetwork/gonode/p2p"
+	"github.com/pastelnetwork/gonode/pastel"
+	"github.com/sbinet/npyio"
+)
+
+const (
+	synchronizationIntervalSec    = 2
+	synchronizationTimeoutSec     = 60
+	runTaskIntervalMin            = 2
```

Comment:
don't really think 2 minutes are reasonable.
maybe a half of hour?
could you double check with Jeff?

Reply:
okay. let me check